core/types/src/error.rs

```diff
@@ -1,9 +1,375 @@
 // Copyright (c) 2022 The MobileCoin Foundation
 
+//! SGX Error types
+
+use core::convert::From;
+use mc_sgx_core_sys_types::sgx_status_t;
+use serde::{Deserialize, Serialize};
```

Comment:
This should be gated on a `"serde"` feature.

Reply:
done